darts/tests/explainability/test_shap_explainer.py

```diff
@@ -267,19 +268,28 @@ def test_explain(self):
         with self.assertRaises(ValueError):
             # wrong horizon
             results.get_explanation(horizon=5, component="price")
+            results.get_feature_values(horizon=5, component="price")
```

Comment:
Each of these checks should be within a dedicated `with self.assertRaises(ValueError)` scope.

Currently, as soon as one check failed it will skip the ones afterwards and exit the `with` scope.

Would you mind fixing this in general in this test file (I see some occurences also in the existing code, not related to your PR) :)

Reply:
haha silly mistake - thx for spotting it. Adjusted it across the whole test file